src/react/components/pages/homepage/homePage.tsx

```diff
@@ -64,9 +65,11 @@ export default class HomePage extends React.Component<IHomePageProps, IHomePageS
         cloudPickerOpen: false,
     };
 
+    private filePicker: React.RefObject<FilePicker> = React.createRef();
     private newProjectRef = React.createRef<HTMLAnchorElement>();
     private deleteConfirmRef = React.createRef<Confirm>();
     private cloudFilePickerRef = React.createRef<CloudFilePicker>();
+    private importConfirm: React.RefObject<Confirm> = React.createRef();
```

Comment:
Maybe here we should use importConfirmRef instead of importConfirm? Since we have deleteConfirmRef on top. In my opinion, it looks more consistent.